src/force/ilp_tmd_sw.cu

```diff
@@ -126,7 +127,7 @@ ILP_TMD_SW::ILP_TMD_SW(FILE* fid_ilp, FILE* fid_sw, int num_types, int num_atoms
   // init constant cutoff coeff
   float h_tap_coeff[8] = \
     {1.0f, 0.0f, 0.0f, 0.0f, -35.0f, 84.0f, -70.0f, 20.0f};
-  cudaMemcpyToSymbol(Tap_coeff_tmd, h_tap_coeff, 8 * sizeof(float));
+  gpuMemcpyToSymbol(Tap_coeff_tmd, h_tap_coeff, 8 * sizeof(float));
   CUDA_CHECK_KERNEL
```

Comment:
What is the role of `CUDA_CHECK_KERNEL` for HIP? should this perhaps be renamed? I couldn't find where it is defined.

Reply:
reamed to GPU_CHECK_KERNEL